src/taipy/config/_json_serializer.py

```diff
@@ -36,8 +41,27 @@ def _read(cls, filename: str) -> _Config:
 
     @classmethod
     def _serialize(cls, configuration: _Config) -> str:
-        return json.dumps(cls._str(configuration), ensure_ascii=False, indent=0, check_circular=False)
+        return json.dumps(
+            cls._str(configuration), ensure_ascii=False, indent=0, check_circular=False, cls=_CustomEncoder
```

Comment:
Since `cls._str(configuration)` already convert `datetime.datetime` and `datetime.timedelta` to string, do we need the encoder here?